server/app.py

```diff
@@ -324,6 +324,12 @@ def cancel_job(job_id):
         JobStatusResponse[status.CANCELED]
     """
     job = process_file.AsyncResult(job_id)
+    input_job_id = job.info["input_job_id"]
```

Comment:
Add checks to make sure this metadata exists, for both input and output

Reply:
it should right. on would it stop existing if the job is done?

Reply:
added the check